app-admin/pinkybar/pinkybar-9999.ebuild

```diff
@@ -0,0 +1,142 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+EGIT_REPO_URI="https://github.com/su8/pinky-bar.git"
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit git-r3 python-single-r1 autotools
+
+DESCRIPTION="Gather some system information and show it in this statusbar program"
+HOMEPAGE="https://github.com/su8/pinky-bar"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="+alsa colours cpp drivetemp drivetemp-light dvd libnl lisp lua mpd ncurses
+	+net +pci perl python2 r ruby sensors slang smartemp tcl weather x11"
+
+DEPEND="
+	sys-devel/autoconf
+	>=sys-devel/automake-1.14.1
+	sys-apps/gawk
+	sys-devel/m4
+	dev-lang/perl
+"
+RDEPEND="
+	alsa? ( media-libs/alsa-lib )
+	drivetemp? ( net-misc/curl app-admin/hddtemp )
+	drivetemp-light? ( app-admin/hddtemp )
+	dvd? ( dev-libs/libcdio )
+	libnl? ( >=dev-libs/libnl-3.2.27 virtual/pkgconfig )
+	lisp? ( dev-lisp/ecls )
+	lua? ( dev-lang/lua )
+	mpd? ( media-sound/mpd media-libs/libmpdclient )
+	ncurses? ( sys-libs/ncurses )
+	net? ( sys-apps/iproute2 )
+	pci? ( sys-apps/pciutils )
+	python2? ( ${PYTHON_DEPS} )
+	r? ( dev-lang/R )
+	ruby? ( dev-lang/ruby )
+	sensors? ( sys-apps/lm_sensors )
+	slang? ( sys-libs/slang )
+	smartemp? ( sys-apps/smartmontools )
+	tcl? ( dev-lang/tcl )
+	weather? ( net-misc/curl app-arch/gzip )
+	x11? ( x11-libs/libX11 )
+
+"
+REQUIRED_USE="
+	drivetemp? ( !smartemp )
+	drivetemp-light? ( !smartemp )
+	ncurses? ( !x11 )
+	smartemp? ( !drivetemp !drivetemp-light )
+	x11? ( !ncurses )
+"
+
+src_prepare() {
+	default
+
+	elog 'Generating Makefiles'
+	perl set.pl 'gentoo' || die
+	eautoreconf -if
+}
+
+src_configure() {
+
+	CONFIGURE_OPTS=(
+		$(use_with alsa)
+		$(use_with colours)
+		$(use_with cpp)
+		$(use_with drivetemp)
+		$(use_with drivetemp-light)
+		$(use_with dvd)
+		$(use_with libnl)
+		$(use_with lisp)
+		$(use_with lua)
+		$(use_with mpd)
+		$(use_with ncurses)
+		$(use_with net)
+		$(use_with pci)
+		$(use_with perl)
+		$(use_with python2)
+		$(use_with r)
+		$(use_with ruby)
+		$(use_with sensors)
+		$(use_with slang)
+		$(use_with smartemp)
+		$(use_with tcl)
+		$(use_with weather)
+		$(use_with x11)
+		api_key='28459ae16e4b3a7e5628ff21f4907b6f'
```

Comment:
What's this key and why is it public?

Reply:
It's my key generated for the **--with-weather** option.

Reply:
Doesn't that violate ToS of the weather services used?

Reply:
As long as the service usage remains low, there shouldn't be a issue.

Reply:
I'm not asking if it 'should not be an issue'. I'm asking if you're violating the agreement with the service provider and — which is way worse — are involving users unknowingly in that.

Reply:
No according to the Licenses section - https://openweathermap.org/terms

Reply:
In section "Viruses, hacking and other offences" the TOS say:
"You must not attempt to gain unauthorised access to our site, the server on which our site is stored or any server, computer or database connected to our site."

I am not sure if using the API key from somebody else would be "unauthorised access", but we should rather stay on the safe side. So, can you ask upstream if such usage complies to their TOS?

Reply:
Dropping the **--with-weather** option as per ![email](https://user-images.githubusercontent.com/29376083/38190584-35e59010-3665-11e8-81ef-274ff8a11753.png)